src/main/java/uk/gov/pay/ledger/agreement/state/AgreementState.java

```diff
@@ -0,0 +1,67 @@
+package uk.gov.pay.ledger.agreement.state;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import uk.gov.pay.ledger.event.model.SalientEventType;
+
+import java.util.Map;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonFormat(shape = JsonFormat.Shape.OBJECT)
+public enum AgreementState {
+
+    CREATED("created"),
+    ACTIVE("active"),
+    CANCELLED("cancelled"),
+    EXPIRED("expired");
+
+    private final String oldStatus;
```

Comment:
If we're not using old status, code or message I think it's OK to leave them out from this -- I think some of the old status pieces for transactions were only used to transition from a previous method of showing statuses to the current system and shouldn't be needed by this

Reply:
👍